scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -209,6 +249,28 @@ def _calculate_pairs(self, rebin):
                      run,
                      rebin=rebin)
 
+    def calculate_all_diffs(self):
+        self._calculate_diffs(rebin=False)
+        if self._do_rebin():
```

Comment:
In all of the code we have this `if rebin` then do it again. This is because we can otherwise end up with the binned and raw data being out of sync. The scientists will often plot the binned data, but fit to the raw. So if we only update one it will cause errors.